tests/h/views/admin/documents_test.py

```diff
@@ -53,6 +53,10 @@ def test_it_errors_if_validation_fails(
     def move_annotations_by_url(self, patch):
         return patch("h.views.admin.documents.move_annotations_by_url")
 
+    @pytest.fixture
+    def views(self, pyramid_request):
+        return DocumentsAdminViews(pyramid_request)
```

Comment:
Introduced this fixture as I wanted it in the get test above. Thought we might as well re-use it